.github/workflows/ci.yml

```diff
@@ -0,0 +1,91 @@
+name: Package and Publish to Visual Studio Marketplace
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+
+      - name: Set Git configuration
+        run: |
+            git config user.name "${GITHUB_ACTOR}"
+            git config user.email "${GITHUB_ACTOR}@bots.github.com"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Bump version and create Git tag
+        uses: Aqovia/npm-version-increment@tidyup
+        id: version
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Generate release changelog
+        uses: heinrichreimer/github-changelog-generator-action@v2.3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          base: CHANGELOG.md
+      
+      - name: Install vsce CLI
+        run: npm install -g @vscode/vsce
+        
+      - name: Extract extensionPack and update README
+        shell: pwsh
```

Comment:
Can you use global shell setting instead?
https://docs.github.com/en/actions/using-jobs/setting-default-values-for-jobs#example-set-the-default-shell-and-working-directory

Reply:
That would save 2 lines of code and use 3. It also hides the shell at the top of the YAML instead of being explicitly shown in the step.

Reply:
We can leave it.